src/plugins/console/public/services/autocomplete.ts

```diff
@@ -53,7 +53,11 @@ export class AutocompleteInfo {
         const collaborator = this.mapping;
         return () => this.alias.getIndices(includeAliases, collaborator);
       case ENTITIES.FIELDS:
-        return this.mapping.getMappings(context.indices, context.types);
+        return this.mapping.getMappings(
+          Object.getPrototypeOf(context),
```

Comment:
Why are we using `Object.getPrototypeOf` here?

Reply:
Because the parent (autocomplete) context we need to update is assigned to a prototype [here](https://github.com/elastic/kibana/blob/6bafb59fd5cc6cac55f07bd42df97e2ece0633e9/src/plugins/console/public/lib/autocomplete/engine.js#L38)